README.md

```diff
@@ -7,20 +7,57 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/simple-icons-font@v2/font/simple-icons.min.css"
+      type="text/css">
+```
+
+### NodeJS
+
+```
+$ npm install simple-icons-font
+```
 
-### General Usage
+After installation, the icon font can then be found in `node_modules/simple-icons-font/font` along with stylesheets for a basic setup.
 
-This package can be installed into your project with NPM to provide all of our icons as an icon font.
+### Manual download
 
-```bash
-npm install simple-icons-font
+1. [Download the latest version of the package][npm-registry-tarball-link], extract it, and move the content of the `font` folder to your desired location.
+1. Reference the CSS file in a `link` tag of your HTML:
+
+```html
+<link rel="stylesheet"
+      href="/path/to/simple-icons.min.css"
+      type="text/css">
 ```
 
-After installation, the icon font can then be found in `node_modules/simple-icons-font/font` along with a stylesheet for
-a basic setup.
+## Usage
+
+The CSS files of simple-icons-font contains next classes:
+
+- `simpleicons`: Adds the font to the container that will display the icon.
+- `simpleicons--color`: Colorizes the icon with the main color of the brand.
+- `simpleicons-[ICON SLUG]`: Replace here `[ICON SLUG]` by with the correspondent slug for the icon. Their slugs correpond to filenames, which you can see listed in [simple-icons/icons][simple-icons--icons-dir-link].
+
+### Example
+
+```html
+<i class="simpleicons simpleicons--color simpleicons-simpleicons"></i>
+```
```

Comment:
I think an example will do to let people know how this works.

    Use any of the icons available in simple-icons by adding the following classes to a node in your HTML. Use the `simple-icons--color` class to apply the brand's color to the icon.

    ```html
    <i class="simpleicons simpleicons-[ICON NAME]"></i>
    <i class="simpleicons simpleicons-[ICON NAME] simple-icons--color"></i>
    ```

    Where `[ICON NAME]` is replaced by the icon name, for example:

    ```html
    <i class="simpleicons simpleicons-simpleicons"></i>
    <i class="simpleicons simpleicons-simpleicons simple-icons--color"></i>
    ```

    In this example we use the `<i>` tag, but any inline HTML tag should work as you expect.


We could still add more detailed documentation if it turns out to be necessary/you feel its necessary, but I would put it after the example. What do you think?

Also, I don't feel like it is necessary to explain that the `[ICON NAME]` should be the slug, at least we don't do it in the main project and it seems most people figure it out. That said, I'm not against adding it. Maybe wait and hope someone opens an issue if they experience difficulty because of this?